.github/workflows/raydp.yml

```diff
@@ -50,9 +50,11 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          pip install wheel
+          pip install pytest koalas tensorflow torch tabulate ray[tune] xgboost_ray
           HOROVOD_WITH_GLOO=1
           HOROVOD_WITH_PYTORCH=1
-          pip install wheel pytest koalas tensorflow torch tabulate ray[tune] xgboost_ray horovod[pytorch,ray]
+          pip install horovod[pytorch,ray]
```

Comment:
what's the difference?

Reply:
install wheel first so that for the rest pip can build their wheels. For horovod I'm not sure, just because once in the CI it says horovod torch is not built. So I try to install it after pytorch is installed. And the running time of this step does vary after this change.

Reply:
OK, thanks.